toolkit/scss/forms/_summary.scss

```diff
@@ -46,7 +49,7 @@
 
 .summary-item-no-content {
   color: $secondary-text-colour;
-  @include core-16;
+  @include copy-16;
```

Comment:
It looks pretty horrible with the different padding.  I could just go back to core-16, as I'm not sure the fallback for tablets would look great anyway with the padding not being great on full screen view.